pkg/plugins/runtime/k8s/controllers/configmap_controller_test.go

```diff
@@ -96,7 +96,8 @@ var _ = Describe("ServiceToConfigMapMapper", func() {
 			Expect(k8sClient.Create(ctx, &givenNamespace)).To(Succeed())
 			givenService.Namespace = givenNamespace.Name
 			Expect(k8sClient.Create(ctx, &givenService)).To(Succeed())
-			for _, pod := range givenPods {
+			for i := range givenPods {
+				pod := givenPods[i]
 				pod.Namespace = givenNamespace.Name
 				Expect(k8sClient.Create(ctx, &pod)).To(Succeed())
```

Comment:
I guess these are kind of spurious, right? In that here for example the `Create` call finishes immediately and so it's not bad to take a pointer, it's just that `gosec` doesn't know we haven't stored it somewhere?

Or is this a real bug?

Doesn't matter either way, definitely easier to just forbid pointers to range variables.

Reply:
Not a real bug but I found that changing the code was better than adding a bunch of lint ignore